src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp

```diff
@@ -417,7 +426,8 @@ void registerInputFormatParquet(FormatFactory & factory)
                     sample,
                     settings,
                     max_parsing_threads,
-                    min_bytes_for_seek);
+                    min_bytes_for_seek,
+                    is_remote_fs);
```

Comment:
No need to batch row groups for readinf from local filesystem, it will be even slower according to my tests

Reply:
Good info.

```suggestion
                    /// No need to batch row groups for reading from local filesystem, it will be even slower according to my tests
                    is_remote_fs);
```